### PR TITLE
Change build files, gradle wrapper properties and rename cameraview module

### PR DESCRIPTION
**Pull request by belanagy**

### CODE REVIEW

File: app/.gitignore [hunk 1]
```diff
@@ -1 +1,10 @@
+*.iml
```
Comment (WonderCsabo) [inline app/.gitignore:1]: Two lines are needed here only:
```
/build
*.iml
```

File: sccameraview/build.gradle [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+allprojects {
+    repositories {
+        jcenter()
+    }
+}
+
+dependencies {
+    compile 'com.android.support:appcompat-v7:25.3.1'
```
Comment (WonderCsabo) [inline sccameraview/build.gradle]: This is not needed.

File: sccameraview/build.gradle [hunk 3]
```diff
@@ -0,0 +1,33 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "26.0.0"
+
+    defaultConfig {
+        minSdkVersion 16
```
Comment (WonderCsabo) [inline sccameraview/build.gradle]: 14

**Review by WonderCsabo — CHANGES_REQUESTED** [review]

File: app/.gitignore (hunk 1 above)
Comment (belanagy) [reply app/.gitignore:1]: done

**Comment by belanagy** [comment]
Changes applied by your suggestions

**Review by WonderCsabo — APPROVED** [review]

**Pull request merged**